src/test/java/ui/TestTerminalUI.java

```diff
@@ -87,19 +87,16 @@ public void testGetInputCharWhenCharIsTyped() throws IOException {
 
 	@Test
 	public void testGetInputCharWhenAnotherCharIsTyped() throws IOException {
-		scanner = new Scanner(new ByteArrayInputStream("a".getBytes()));
+		terminal = settingInputStream("a", 4);
 
-		terminal = new TerminalUI(scanner, 4);
 		char returned = terminal.getInputChar();
 
 		assertThat(returned).isEqualTo('a');
 	}
 
 	@Test
 	public void testGetInputCharWhenSeveralCharsAreTyped() throws IOException {
-		scanner = new Scanner(new ByteArrayInputStream("abc".getBytes()));
-
-		terminal = new TerminalUI(scanner, 4);
+		terminal = settingInputStream("a", 4);
```

Comment:
Meglio mettere in input diversi caratteri come dice il test:
`terminal =  setttingInputStream("abc", 4);`